src/EGraphs/egraph.jl

```diff
@@ -68,7 +74,15 @@ function addparent!(@nospecialize(a::EClass), n::VecExpr, id::Id)
   push!(a.parents, (n => id))
 end
 
+"""
+    merge_analysis_data!(a::EClass{D}, b::EClass{D})::Tuple{Bool,Bool,Union{D,Nothing}} where {D}
+
+Returns a tuple of: `(did_update_a, did_update_b, newdata)` where:
```

Comment:
Please specify that this is an internal function and should not be called directly by Metatheory.jl users